docs/source/whatsnew/0.8.4.txt

```diff
@@ -38,7 +38,8 @@ None
 Maintenance and Refactorings
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-None
+* Asset databases now contain version information to ensure compatibility
+  with current Zipline version.
```

Comment:
@llllllllll I added a whatsnew in its own commit - does that follow the PR dance you mentioned?